lib/repositories/competition/competition_screen_repository.dart

```diff
@@ -7,6 +7,8 @@ class CompetitionScreenRepository {
 
   final BrowsingRemoteService browsingRemoteService;
 
+  final statusListing = "1H-HT-2H-ET-FT";
```

Comment:
You have the same `statusListing` variable in `app_const_variables`

Reply:
I forgot to remove it since its not used anymore, fixed